test/ReCaptchaTest.php

```diff
@@ -257,30 +257,22 @@ public function testVerifyWithMissingSecretKey()
     {
         $this->expectException('ZendService\\ReCaptcha\\Exception');
 
-        $this->reCaptcha->verify('challenge', 'response');
+        $this->reCaptcha->verify('response');
     }
 
     public function testVerifyWithMissingIp()
     {
         $this->expectException('ZendService\\ReCaptcha\\Exception');
 
         $this->reCaptcha->setSecretKey($this->secretKey);
-        $this->reCaptcha->verify('challenge', 'response');
-    }
-
-    public function testVerifyWithMissingChallengeField()
```

Comment:
This test is now removed as verify function has only one parameter - `responseField`.